settings.gradle.kts

```diff
@@ -45,7 +45,7 @@ rootProject.name = "spark-android"
 
 include(":spark")
 // Disable sample app for now, until we have something interesting to showcase
-// include(":sample")
+include(":sample")
```

Comment:
We should let it be commented and uncomment it when we'll distribute it imo

Reply:
We should discuss it further. @SimonMarquis and me we are for including the sample app. This would help to avoid possible build issues and bugs in the future. However, we can comment again later if we see that there is no practical usage indeed.